Readme.rst

```diff
@@ -69,8 +69,7 @@ To commit changes to the framework switch into the Framework directory (``cd Pac
 
 In the root directory of the development distribution, you can do the following things:
 
-To run tests, run ``./bin/phpunit -c ./Build/BuildEssentials/PhpUnit/UnitTests.xml`` for unit or ``./bin/phpunit -c ./Build/BuildEssentials/PhpUnit/FunctionalTests.xml`` for functional/integration tests. If you are on 6.0 or later, you can also run ``./bin/psalm --config=Packages/Framework/psalm.xml``
-to run static analysis tools.
+To run tests, run ``./bin/phpunit -c ./Build/BuildEssentials/PhpUnit/UnitTests.xml`` for unit or ``./bin/phpunit -c ./Build/BuildEssentials/PhpUnit/FunctionalTests.xml`` for functional/integration tests and for static analysis ``composer lint``.
```

Comment:
Those commands will not work "the same way", as for the`composer lint` call you need to `cd Packages/Framework` first, no? Or, if you are "there", the PhpUnit calls will not work, as `Build/BuildEssentials` doesn't exist.